src/components/select-language/select-language.tsx

```diff
@@ -232,6 +232,7 @@ export const langToFlagIconMap: Record<string, string> = {
   "yo": "ng", // Yoruba (Nigeria)
   "zu": "za", // Zulu (South Africa)
   "xh": "za", // Xhosa (South Africa)
+  "vi": "vn", // Vietnam (South East Asia)
```

Comment:
Hello! 
Thanks for attempt to localise extension to Vietnamese, but you forgot most important pat: localization strings itself that must be in `_locales/vi/messages.json`.
Please copy-paste folder `_locales/en`, rename and update all messages there to your local language.

Reply:
Hello!
Thanks for replying me. I saw that you already made the Fix and I also added the translation of Vietnamse. Let me know if you need me to change something else.

Reply:
All good now! Thanks for translation!